docs/cmd/generate_changelog_doc.go

```diff
@@ -194,24 +237,293 @@ func getAllReleases(client *github.Client, repo string) ([]*github.RepositoryRel
 	if err != nil {
 		return nil, err
 	}
+
+	if sortedByVersion {
+		sort.Slice(allReleases, func(i, j int) bool {
+			releaseA, releaseB := allReleases[i], allReleases[j]
+			versionA, err := versionutils.ParseVersion(releaseA.GetTagName())
+			if err != nil {
+				return false
+			}
+			versionB, err := versionutils.ParseVersion(releaseB.GetTagName())
+			if err != nil {
+				return false
+			}
+			return Version(*versionA).LessThan(Version(*versionB))
```

Comment:
why not,
```suggestion
			return versionB.MustIsGreaterThan(*versionA)
```
and avoid re-declaring types for `Version` and `Versions` at the bottom of the file?

Reply:
The change is valid. However, I'm using the `sort` library to sort versions, and to sort on a type requires that the type have a `LessThan` function.